lib/node/api.js

```diff
@@ -985,31 +1013,11 @@ api.getTransactionHistory = async (req, res) => {
     const txids = (await btcClient('getaddresstxids', [address])).result;
     const mempoolTx = (await btcClient('getaddressmempool', [{ addresses: [address] }], isOmni)).result;
     const txs = txids.concat(mempoolTx.map((x) => x.txid));
-    const allTxDetails = [];
     const txDetails = await getTransactionDetails(txs);
     const promises = txDetails.map(async (txDetailParam) => {
```

Comment:
Can this `async` keyword be removed as well now?

Reply:
Fixed here: https://github.com/imthe-1/multi-dimension-identity-protocol/pull/4/files/c2c09fb7cdf6ce9928e8381e61347c6d30d69f3c..f96fa1ac8fedb5403059bd5121c9a430c3ae36c2